src/env/emulated_syscalls/linux_syscalls.cpp

```diff
@@ -776,6 +778,102 @@ FunctionCallback::return_t sys_linux_exit(
     return std::monostate();
 }
 
+// ssize_t getrandom(void *buf, size_t buflen, unsigned int flags);
+FunctionCallback::return_t sys_linux_getrandom(
+    MaatEngine& engine,
+    const std::vector<Value>& args
+)
+{
+    addr_t buf = args[0].as_uint(*engine.vars);
+    size_t buflen = args[1].as_uint(*engine.vars);
+    cst_t res;
+
+    // TODO: generate proper random bytes, if needed
```

Comment:
```suggestion
    // TODO: generate proper random bytes, if needed. Since the symbolic execution process
    // must be deterministic and reproducible, we must use a deterministic PRNG.
```
Just adding a note to not forget about the need for determinism when emulating randomness